cmd/install/assets/hypershift_operator.go

```diff
@@ -460,6 +461,13 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 		})
 	}
 
+	if o.EnableSizeTagging {
```

Comment:
should we also be setting ENABLE_SIZE_TAGGING to 0 if its not enabled ? Or will a new deployment of the HO remove all the previously set envars?

Reply:
Anything other than it being present and set to "1" is considered false, so I don't think it needs to be set.